basketball_reference_web_scraper/client.py

```diff
@@ -212,6 +212,35 @@ def team_box_scores(day, month, year, output_type=None, output_file_path=None, o
     )
     return output_service.output(data=values, options=options)
 
+def get_roster(team, year=None, output_type=None, output_file_path=None, output_write_option=None, json_options=None):
+    try:
+        http_service = HTTPService(parser=ParserService())
+        if year == None:
+            today = datetime.now()
+            year = today.year
+            if today.month >=7:
+                year += 1
+        if len(team) > 3:
+            team=TEAM_TO_TEAM_ABBREVIATION[team.upper()]
+        values=http_service.get_team_roster(team=team, year=year)
+    except requests.exceptions.HTTPError as http_error:
+        if http_error.response.status_code == requests.codes.not_found:
+            raise InvalidTeam(team=team, year=year)
+        else:
+            raise http_error
+
+    options = OutputOptions.of(
+        file_options=FileOptions.of(path=output_file_path, mode=output_write_option),
+        output_type=output_type,
+        json_options=json_options,
+        csv_options={"column_names": "Players"}
```

Comment:
Here's the `Roster` table

<img width="552" alt="image" src="https://github.com/user-attachments/assets/b7ca83c5-9043-45c3-9a3b-2a2a3323e60f">

Happy to add more values in the future, but at a minimum the values should be

* `name`
* `slug` (or `player_id`)